app/Http/Controllers/PostController.php

```diff
@@ -25,9 +27,11 @@ public function __construct(
     public function index()
     {
         $posts = $this->index_post_service->execute();
+        $tags = $this->index_tag_service->execute();
```

Comment:
コンストラクタインジェクションが増えてきているので、
```suggestion
    public function index(
        private IndexPostServiceInterface $index_post_service,
        private IndexTagServiceInterface $index_tag_service
     ) {
        $posts = $index_post_service->execute();
        $tags = $index_tag_service->execute();
```
な感じのメソッドインジェクションでもいいかもですね。

Reply:
@SardineTa23 
なるほど、Provider の挙動が分かっていませんでしたが、普通のメソッドでも注入できるんですね。
その形式で行います！